### ISSUE TITLE
Setting to limit number of packets log entries

### ISSUE THREAD

**Issue by jorik392** [henrypp/simplewall#941]
Please add a setting to limit packets log entries to user customizable number of entries. Currently, if you click the packets log tab and there are too many entries the program freezes.

**Comment by henrypp** [comment]
Ok

**Comment by henrypp** [comment]
done

**Comment by henrypp** [comment]
No option. For now it is only realised in-place and limited by 900 items. I think this is enough by now.

**Comment by jorik392** [comment]
I see. Can this issue be reopened if you think this has a chance to be implemented in the future? I think a setting for this that the user can change would be nice depending on preference to show more or less number of entries or even by time like showing only entries in the last 1 hour or 24 hours.

**Comment by henrypp** [comment]
Yes it can be reopened, but implementation is depends on requests.

**Comment by jorik392** [comment]
@henrypp I'm not able to reopen. Can you reopen issue?

**Comment by henrypp** [comment]
@jorik392 why?
